### ISSUE TITLE
Hide config file

### ISSUE THREAD

**Issue by adaur** [featherbb/featherbb#74]

**Comment by peloyeje** [comment]
We could use a specific htaccess in the include folder to hide the config.json file, such as : 

```
<Files ~ "\.json$">
 Order allow,deny
 Deny from all   
 Satisfy All     
</Files>
```

... And provide a specific nginx/lighttpd config in the documentation

Or we could turn config.php into a dotted file
Or we could store the config in a php array (like before)

**Comment by adaur** [comment]
The old way seems more universal

**Comment by peloyeje** [comment]
Solved in feede8259befb44a712f6615e5b86987f110c269
